.github/workflows/pre-commit-check.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       run: |
         brew install tfenv tflint terraform-docs pre-commit
         pre-commit install
-        tfenv install
+        tfenv install min-version
```

Comment:
I dunno what you were trying with this?

Reply:
To use the below instead of the `.terraform-version` like we discussed
```
terraform {
  #   required_version  = "~> 0.12.0"
  required_version = ">= 0.12.25"
}

```